Cargo.toml

```diff
@@ -29,6 +29,7 @@ macro_railroad_annotation = "1.0.3"
 test-strategy = "0.3.1"
 proptest = "1.2.0"
 criterion = { version = "0.5.1", features = ["html_reports"] }
+miette = { version = "7.2.0", features = ["fancy"] }
```

Comment:
probably don't want any features enabled in the root-level cargo.toml?

Reply:
Ah, I wasn't sure about whether we needed to include it here so that it would be available downstream. Apparently not. I'll fix that.